user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/SLCloud_xml.md

```diff
@@ -25,3 +25,6 @@ From DataMiner 10.2.0/10.1.5 onwards, you can configure the timeout of the conne
     ```
 
 4. Save your changes and restart DataMiner.
+
+> [!NOTE]
+> From DataMiner 10.3.11/10.3.0 [CU8] onwards<!--RN 37401-->, when DataMiner makes changes to the *SLCloud.xml* file, the old version of that file is saved in the *C:\Skyline DataMiner\Recycle Bin* folder.
```

Comment:
I've reworded the notes slightly so that this is more clear, @LaurensVergote. Thanks for the feedback!